src/main/client/src/components/app/side-bar/map-layers/map-layers-active/MapLayersActiveView.js

```diff
@@ -32,31 +33,51 @@ const MapLayersView = ({
             <Droppable droppableId="droppable">
```

Comment:
Droppable setup issue [droppableId: "droppable"]:DroppableProvided > placeholder could not be found.Please be sure to add the {provided.placeholder} React Node as a child of your Droppable.More information: https://github.com/atlassian/react-beautiful-dnd/blob/master/docs/api/droppable.md
    in Droppable (created by Connect(Droppable))
    in Connect(Droppable) (at MapLayersActiveView.js:33)
    in MapLayersView (at MapLayersActive.js:104)
    in MapLayersActive (created by ConnectFunction)
    in ConnectFunction (at MapLayersView.js:144)
    in MapLayersView (at MapLayers.js:80)

Reply:
Tämän ei pitäisi tapahtua enää, kun tehtävässä on muutettu react-beautiful-dnd kirjaston versiota.